frontend/src/components/MainHeader.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="main-header">
+    <header class="header">
+      <div class="container">
+        <a target="_blank" href="https://woowacourse.github.io/"
+          ><img class="logo" src="/assets/logo/logo_full_dark.png"
+        /></a>
+        <h3 class="title"><router-link to="/recruits">지원</router-link></h3>
+      </div>
+    </header>
+  </div>
+</template>
+
+<script>
+export default {}
+</script>
+
+<style scoped>
+.main-header {
+  margin-bottom: 80px;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 80px;
+  background: #fff;
+  border-bottom: 1px solid #f1f1f1;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  width: 800px;
+}
```

Comment:
요친구에도 슥 넣어볼까용

![image](https://user-images.githubusercontent.com/37579660/94873505-569c1680-048a-11eb-869f-3025d688e694.png)

```suggestion
.container {
  display: flex;
  align-items: center;
  width: 800px;
  user-select: none;
}
```